src/main/java/de/hpi/bpt/argos/core/ArgosImpl.java

```diff
@@ -49,8 +62,12 @@ public void start() {
 			return;
 		}
 
-		// TODO: parse static data
-		EventTypeParserImpl.getInstance().loadEventTypes();
+		WatchImpl.start("starting argos backend", () -> {});

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either remove or fill this block of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS00108)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') '{' is not followed by whitespace. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=checkstyle%3AWhitespace_Around_BP2016)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') '}' is not preceded with whitespace. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=checkstyle%3AWhitespace_Around_BP2016)